package.json

```diff
@@ -27,20 +27,21 @@
     "browser-date-formatter": "^2.1.1",
     "browser-sync": "^2.18.12",
     "browserify-middleware": "^8.0.0",
+    "clipboard": "^1.7.1",
     "compression": "^1.7.1",
     "connect-browser-sync": "^2.1.0",
     "connect-slashes": "^1.3.1",
     "cookie-parser": "^1.4.3",
     "cross-env": "^5.1.1",
     "dotenv": "^4.0.0",
-    "electron-apps": "1.1976.0",
+    "electron-apps": "^1.1976.0",
```

Comment:
can you revert these version updates that aren't related to the PR? That will help avoid merge conflicts, as electron-apps was just updated in a more recent PR that landed.

Reply:
I ran the command `npm install` (or `npm update` I don't remember) and it automatically added this. I will remove this change.